src/libcore/lib.rs

```diff
@@ -167,6 +167,7 @@ mod core {
     pub use marker;
     pub use option;
     pub use iter;
+    pub use default;
```

Comment:
Looks like it's not.

On Wed, May 27, 2015 at 3:30 PM, Alex Crichton <notifications@github.com>
wrote:

> In src/libcore/lib.rs
> <https://github.com/rust-lang/rust/pull/25817#discussion_r31187466>:
>
> > @@ -167,6 +167,7 @@ mod core {
> >      pub use marker;
> >      pub use option;
> >      pub use iter;
> > +    pub use default;
>
> Is this still necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/25817/files#r31187466>.
>